input/images-source/ContentExchangeBasic.plantuml

```diff
@@ -0,0 +1,42 @@
+@startuml
+autonumber
+
+actor Clinician
+participant "App1\n(Subscriber)" as App1
+participant Hub
+participant "App2\n(Subscriber)" as App2
+
+=== Open diagnostic report ===
+
+Clinician -> App1: Open Report
+activate App1
+App1 -> Hub++: DiagnosticReport-open
+Hub -> Hub: Store Context and Generate VersionId
+App1 <-- Hub--: 202 Accepted
```

Comment:
Removed the autonumber and did not add them as the original image did not have them as well.

Reply:
Updated as suggested

Reply:
Since we now have the fancy PlantUML language available to us, I propose the following PlantUML (which produces essentially the same diagram with the latest comments with a few syntactical tweaks.

@startuml
skinparam shadowing false

 actor Clinician
 participant "App1\n(Subscriber)" as App1
 participant Hub
 participant "App2\n(Subscriber)" as App2

 Clinician -> App1: Open Report
 group FHIRcast: Open Report
 activate App1
 App1 -> Hub: DiagnosticReport-open
 activate Hub
 Hub -> Hub: Store Context and Generate VersionId
 App1 <-- Hub: 202 Accepted
 |||
 deactivate App1
 Hub -\ App1: DiagnosticReport-open
 rnote right App1: with versionId
 Hub -\ App2: DiagnosticReport-open
 rnote left App2: with versionId
 deactivate Hub
end
...
Clinician -> App2: Make Observation
group FHIRcast: Make Observation
activate App2
 App2 -> Hub: DiagnosticReport-update
 activate Hub
 note right Hub: with versionId and Observation
 Hub -> Hub: Update Content and generate new VersionId
 App2 <-- Hub: 202 Accepted
 deactivate App2
 |||
 Hub -\ App1: DiagnosticReport-open
 note right App1: with Observation, versionId,\nand priorVersionId
 Hub -\ App2: DiagnosticReport-open
 note left App2: with Observation, versionId,\nand priorVersionId
 deactivate Hub
end
...
Clinician -> App2: Close report
group FHIRcast: Close Report
activate App2
App2 -> Hub: DiagnosticReport-close
activate Hub
App2 <-- Hub: 202 Accepted
deactivate App2
|||
Hub -\ App1: DiagnosticReport-close
Hub -\ App2: DiagnosticReport-close
Hub -> Hub: Purge Context including Content
end
deactivate Hub

 @enduml